nixos/modules/virtualisation/lxc.nix

```diff
@@ -62,6 +80,8 @@ in
     environment.etc."lxc/lxc.conf".text = cfg.systemConfig;
     environment.etc."lxc/lxc-usernet".text = cfg.usernetConfig;
     environment.etc."lxc/default.conf".text = cfg.defaultConfig;
+    environment.etc."default/lxc".text = cfg.bridgeConfig;
```

Comment:
I think we want this in:
```suggestion
    environment.etc."lxc/lxc-net".text = cfg.bridgeConfig;
```
https://github.com/lxc/lxc/blob/a1d2ec8657e52c4efd2a8d018f2f2f0d5e94afb9/config/init/common/lxc-net.in#L8C1-L8C39

If that doesn't work, try setting [distrosysconfdir](https://github.com/lxc/lxc/blob/a1d2ec8657e52c4efd2a8d018f2f2f0d5e94afb9/meson_options.txt#L121) in [mesonFlags](https://github.com/NixOS/nixpkgs/blob/57610d2f8f0937f39dbd72251e9614b1561942d8/pkgs/by-name/lx/lxc/package.nix#L56)